conf/locale/en/LC_MESSAGES/django.po

```diff
@@ -2699,6 +2685,12 @@ msgstr "Questions about your pleas?"
 msgid "Need help using this service?"
 msgstr "Need help using this service?"
 
+#~ msgid "do not reply to it or click any links"
```

Comment:
I've pushed the change. It's also highlighted that I've missed some changes from #312 leaving `# TODO` markers in. I'll add them to this PR and change the title.